dev/com.ibm.ws.org.apache.cxf.cxf.rt.ws.policy.3.2/src/org/apache/cxf/ws/policy/PolicyOutInterceptor.java

```diff
@@ -134,7 +144,6 @@ protected void handle(Message msg) {
                 for (Assertion a : assertions) {
                     PolicyUtils.printPolicyComponent(a, buf, 1);
                 }
-                LOG.finest(buf.toString());
```

Comment:
Why did you remove this?  Now we don't even call LOG.finest?

Reply:
It seemed redundant to me log the buffer immediately after calling `PolicyUtils.printPolicyComponent(...)`

Reply:
But on second glance that's not been added by Syed's changed, which I thought it was, so I will readd it.